src/material/tooltip/tooltip.ts

```diff
@@ -235,7 +240,8 @@ export class MatTooltip implements OnDestroy, AfterViewInit {
   }
 
   /** Manually-bound passive event listeners. */
-  private _passiveListeners = new Map<string, EventListenerOrEventListenerObject>();
+  private readonly _passiveListeners:
```

Comment:
I'm not sure why this was changed to be an array. I suppose the memory consumption is a few bytes lower, but we'd have to destructure the sub-arrays any time we're iterating over it, and it makes things harder to read. Since this is fairly small (I think ~6 items in the worst case) we should be fine with using a `Map`.

Reply:
It's easier to append to the existing array (since the events are added to it in 2 places) than it would be to add multiple additional entries to the map.

Also it's a bit safer in the case that we somehow get multiple events for the same key, we would not leak them.

And the memory savings. And it's probably a smidge faster as well.

Reply:
I don't feel strongly about `Map` vs tuple type, though you could always merge maps like `merged = new Map([...m1, ...m2])`